internal/kubehelper/client.go

```diff
@@ -11,8 +11,15 @@ import (
 
 func createKubeClient(timeout int64) (*kubernetes.Clientset, error) {
 	var cliKubeconfig string
-	flag.StringVar(&cliKubeconfig, "kubeconfig", "", "absolute path to the kubeconfig file")
-	flag.Parse()
+	for i := 1; i < len(os.Args)-1; i++ {
+		if os.Args[i] == "--kubeconfig" {
```

Comment:
This is incorrect: I can do `kubectl --kubeconfig=/path/to/file` too, so you'll also need to match that.  Luckily, it's not a prefix match, so we don't need to worry about `kubectl --kubeconfi /path/to/file` (note missing `g`).

Confirmed by testing with `kubectl --kubeconfig /dev/null config current-context` (and  variations) and noting the difference in error messages.

Reply:
> This is incorrect: I can do `kubectl --kubeconfig=/path/to/file` too, so you'll also need to match that.

Thanks for catching that! I've updated the PR; please review again!